### PR TITLE
Test for getting a user's pushers

### PR DESCRIPTION
**Pull request by dbkr**
Tests https://github.com/matrix-org/synapse/pull/716

### CODE REVIEW

File: tests/61push/06_get_pusher.pl [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+   check => sub {
+      my ( $alice ) = @_;
+
+      my $profile_tag = "tag";
+      my $app_id = "sytest";
+      my $app_display_name = "SyTest";
+      my $device_display_name = "A testing machine";
+      my $pushkey = "This is my pushkey";
+      my $lang = "en";
+      my $customdata = "This is some custom data";
```
Comment (leonerd) [inline tests/61push/06_get_pusher.pl]: Surely "a custom datum"  ;)

File: tests/61push/06_get_pusher.pl [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+         do_request_json_for( $alice,
+            method  => "GET",
+            uri     => "/r0/pushers",
+         );
+      })->then( sub {
+         my ( $body ) = @_;
+
+         log_if_fail "Get pusher response body", $body;
+
+         assert_json_keys( my $notification = $body, ("pushers"));
```
Comment (leonerd) [inline tests/61push/06_get_pusher.pl]: `qw()`

**Comment by leonerd** [comment]
LGTM besides the `qw()`

File: tests/61push/06_get_pusher.pl (hunk 1 above)
Comment (dbkr) [inline tests/61push/06_get_pusher.pl]: 48c

**Pull request merged**